context-py/src/lib.rs

```diff
@@ -80,6 +81,14 @@ fn junit_parse(xml: Vec<u8>) -> PyResult<Vec<junit::bindings::BindingsReport>> {
         .collect())
 }
 
+#[gen_stub_pyfunction]
+#[pyfunction]
+fn bin_parse(bin: Vec<u8>) -> PyResult<Vec<junit::bindings::BindingsReport>> {
```

Comment:
I want to decouple the bindings report from quick_junit, but that will be handled in a later task.